### ISSUE TITLE
Initial transfer learning neural net

### ISSUE THREAD

**Issue by Olivia-Roscoe** [alewan/capstone#1]
**Obj**: Create an initial trained classifier network using transfer learning with AlexNet or resNet to test against AWS

**AC (Acceptance Criteria):**
1. Output - final test accuracy
2. Input - image to be classified
3. Checkpoint of trained net
4. Built using pytorch
